README.md

```diff
@@ -8,6 +8,39 @@ functional** and **fast User Interface/Experience** (UI/UX)
 with **minimal CSS**
 in **much less time**._
 
+<!-- START doctoc -->
```

Comment:
Why will also test to see whether we need a title here. I thought we would but previewing it, actually looks fine!